### ISSUE TITLE
some questions about method and result

### ISSUE THREAD

**Issue by YiZhuo-Xu** [Chiangsonw/CaLa#5]
I am trying to reproduce the results from the paper, and my training parameters are as follows:
CUDA_VISIBLE_DEVICES='0' python src/blip_fine_tune.py --dataset FashionIQ \
	--num-epochs 30 --batch-size 64 \
	--max-epoch 15 --min-lr 0 \
	--learning-rate 5e-6 \
	--transform targetpad --target-ratio 1.25 \
	--save-training --save-best --validation-frequency 1 \
	--encoder multi \
	--encoder-arch blip2 \
	--cir-frame sum \
	--tac-weight 0.45 \
	--hca-weight 0.1 \
	--embeds-dim 768 \
But there is a certain gap between my results and those published in the paper. Are there any possible issues that I should pay attention to? Thank you very much! 
Additionally, I noticed that the text input in the Qformer code seems not to be entirely consistent with the flowchart in the paper. Could this potentially have an impact?

**Comment by YiZhuo-Xu** [comment]
Part of my validation result:
13,38.62171471118927,62.617748975753784,44.16114091873169,66.24171137809753,38.420018553733826,57.70363211631775,40.40095806121826,62.18769749005636,51.29432777563731
14,38.27466666698456,64.00594711303711,43.65119934082031,66.13972187042236,38.81256282329559,58.58684778213501,40.24614294370016,62.91083892186483,51.57849093278249
15,37.382251024246216,63.411009311676025,42.937275767326355,65.9357488155365,38.861629366874695,58.881258964538574,39.72705205281576,62.74267236391703,51.234862208366394
16,38.07635009288788,63.95636796951294,43.44722032546997,66.4966881275177,38.81256282329559,58.58684778213501,40.11204441388448,63.013301293055214,51.56267285346985
17,37.679722905159,62.2211217880249,43.19224953651428,65.98674058914185,38.861629366874695,57.70363211631775,39.91120060284933,61.970498164494835,50.94084938367208
18,38.17550837993622,63.361430168151855,43.498215079307556,64.86486196517944,38.27281594276428,57.948970794677734,39.98217980066935,62.05842097600301,51.02030038833618

**Comment by YiZhuo-Xu** [comment]
I made a mistake, setting the learning rate to 1e-5 would yield better results.

**Comment by Chiangsonw** [comment]
Since the fashionIQ has less samples, maybe the higher initial learning rate is better. Thank you for your interest in this project.
